types/sanctuary/sanctuary-tests.ts

```diff
@@ -1,10 +1,11 @@
 /// <reference types="node"/>
-import * as assert from "assert";
 import { create, env } from "sanctuary";
 
 const checkTypes = process.env["NODE_ENV"] !== "production";
 const S = create({checkTypes, env});
 
-assert.equal(S.map(S.concat('@'))(['foo', 'bar', 'baz']), ["@foo", "@bar", "@baz"]);
-assert.equal(S.reduce(S.add)(0)([1, 2, 3, 4, 5]), 15);
-assert.equal(S.flip(S.concat)('foo')('bar'), "barfoo");
+// $ExpectType number
+S.reduce(S.add)(0)([1, 2, 3, 4, 5]);
+
+// $ExpectType string
+S.flip(S.concat)('foo')('bar');
```

Comment:
Did you mean to drop the `S.map` expression?

Reply:
Yeah, it compiled to a really weird expression so it felt very strange to type check it. I'm guessing that it might actually be a bug that it didn't compile to `string[]`, but I didn't have time to look into it.

Since it didn't test anything before (`assert.equal` accepted `any`) I didn't feel like anything was lost by just dropping it...

What do you think?

Reply:
Dropping it seems reasonable to me. Thanks for the explanation.